src/OpenSage.Game/Gui/Apt/SpriteItem.cs

```diff
@@ -345,7 +346,7 @@ public override void RunActions(TimeInterval gt)
 
         public override bool HandleInput(Point2D mousePos, bool mouseDown)
         {
-            foreach (var item in Content.Items.Values)
+            foreach (var item in Content.Items.Values.Reverse())
```

Comment:
Buttons most be handled from top to bottom objects (in terms of depth)

Reply:
It would be better to store these in the correct order rather than reversing the order every time?

Reply:
The order is correct. The rendering happens exactly the opposite way, from far to near and happens much more frequently than the input handling :)

Reply:
And then can you store both directions, so it doesn’t need to be computed every frame?

Reply:
Yep, i’ll do that tomorrow

Reply:
Fixed